src/Components/IssueControl.test.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import {render, screen} from '@testing-library/react'
+import {MockRoutes} from '../BaseRoutesMock.test'
+import {IssuesNavBar, Issues} from './IssuesControl'
+import {act, renderHook} from '@testing-library/react-hooks'
+import useStore from '../store/useStore'
+
+
+const MOCK_ISSUES = [
+  {
+    embeddedUrl: 'url = http://localhost:8080/share/v/p/index.ifc#c:-141.9,72.88,21.66,-43.48,15.73,-4.34',
+    index: 0,
+    id: 10,
+    number: 1,
+    title: 'TEST_ISSUE_TITLE_1',
```

Comment:
can you use a different title and body throughout?  I suggest 2 sentences with a few words each.  These will live with us for a while and will help us test preview, etc. later on